app/services/medical_copays/vbs/service.rb

```diff
@@ -43,7 +44,11 @@ def initialize(opts)
       def get_copays
         raise InvalidVBSRequestError, request_data.errors unless request_data.valid?
 
-        response = request.post("#{settings.base_path}/GetStatementsByEDIPIAndVistaAccountNumber", request_data.to_hash)
+        response = if Flipper.enabled?(:debts_cache_vbs_copays_empty_response)
```

Comment:
Don't forget to add the feature flag to the features.yml file.

`config/features.yml`

You'll see an example from when we added the caching for debts there on line 317